src/map/script_map.cpp

```diff
@@ -1201,6 +1214,8 @@ void MapCclRegister()
 
 	lua_register(Lua, "DefineTileset", CclDefineTileset);
 	lua_register(Lua, "GenerateExtendedTileset", CclGenerateExtendedTileset);
+	lua_register(Lua, "SaveTilesetGraphics", CClSaveTilesetGraphics);
```

Comment:
Prepending by `DEBUG_` is envisageable to enforce it is Debug purposes only.

Reply:
Fixed: 
https://github.com/Wargus/stratagus/pull/732/commits/65b614becd179415aff3a7adb70bccfa0ea62737

Reply:
I didn't meant to protect by `DEBUG` which might be risky from script side.
depending of engine, script might fail...
my proposal was:
```
lua_register(Lua, "DEBUG_SaveTilesetGraphics", CClSaveTilesetGraphics);
```
but that is OK.